### PR TITLE
Create COPs section and add Book Club to Cloud and Engineering

### PR DESCRIPTION
**Pull request by reub-the-cube**
This PR creates a new folder in the root for all Communities of Practice. I'm putting the C&E COP into it, along with the Book Club 'guild' / 'sub-community of interest'. It feels extensible for other [new] COPs and other sub-communities e.g. Delivery, CARE, ED&I, Marketing, Cyber etc (if we wanted to go there).

Why?
I wanted a place we could be transparent about our Book Club and there didn't seem a suitable place (short of create a new Wiki / repo, which is OK too)

I'm looking for feedback on:
- the PR as always!
- at a high level if this tree structure feels suitable
- whether there's a better place for Book Club to live
- at a low level if the starting content for the Book Club is appropriate

PS - CI builds seem pretty flaky on testing links :(

### CODE REVIEW

**Review by owenblacker — COMMENTED** [review]
(Fellow newbie, so feel free to ignore my thoughts, obv)

Broadly, this makes sense to me, but I'd worry that `library.md` would end up getting _very_ big pretty quickly. I'd be tempted to split those 3× h2 sections into separate pages, personally.

File: communities-of-practice/cloud-and-engineering/book-club/welcome.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Purpose
+
+As engineers in delivery teams, we want to continously improve our capabilities and how we deliver working software. By using a technical book to give our minds some direction, we will be able to share our own relevant knowledge and experiences and discuss the book's content - the questions it asks of us - and how it can make us more well-rounded engineers.
+
+## Format?
+
+- It should be a free-flowing session and we'll use the book to provide direction.
+- At the end, we'll have a chat about what to read before the next meet-up.
+- We meet on the third Friday of the month at 1600.
+- It's an open invite to anyone at Made Tech. Just head over to the #eng-book-club Slack channel to pick up the link!
```
[the remark this answers is not in the document]
Comment (reub-the-cube) [reply communities-of-practice/cloud-and-engineering/book-club/welcome.md:16]: Great spot, thanks! I feel renaming the channel might be better, but will canvass some opinion!

[the remark this answers is not in the document]
Comment (reub-the-cube) [reply communities-of-practice/cloud-and-engineering/book-club/welcome.md:16]: We'll rename the channel but I don't seem to be able to work it out so will get it done!

File: communities-of-practice/cloud-and-engineering/book-club/library/books_we_recommend.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Books we recommend
+
+| Title | Author | Recommended By |
+| --- | --- | --- |
+| [Accelerate](https://www.goodreads.com/en/book/show/35747076-accelerate) | Nicole Forsgren, Jez Humble and Gene Kim | Reuben |
+| [Domain-Driven Design](https://www.goodreads.com/book/show/179133.Domain_Driven_Design) | Eric Evans | Reuben |
```
[the remark this answers is not in the document]
Comment (reub-the-cube) [reply communities-of-practice/cloud-and-engineering/book-club/library/books_we_recommend.md:6]: As discussed, perhaps a future improvement, but we want to make it inclusive for members of the public reading the handbook.

**Review by owenblacker — APPROVED** [review]

**Review by jamiem-code — APPROVED** [review]
Really like the idea of adding COP details to the handbook!

File: communities-of-practice/cloud-and-engineering/book-club/welcome.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (owenblacker) [reply communities-of-practice/cloud-and-engineering/book-club/welcome.md:16]: Chasey has renamed the channel 🙂

**Pull request merged**